veps/sig-compute/10-dra-devices/vep.md

```diff
@@ -0,0 +1,781 @@
+# VEP #10: Support DRA devices in KubeVirt
+
+## Release Signoff Checklist
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [x] (R) Enhancement issue created, which links to VEP dir in [kubevirt/enhancements] (not the initial VEP PR)
+
+## Overview
+
+This proposal is about adding support for DRA (dynamic resource allocation) in KubeVirt.
+DRA allows vendors fine-grained control of devices. The device-plugin model will continue
+to exist in Kubernetes, but DRA will offer vendors more control over the device topology.
+
+## Motivation
+
+DRA adoption is important for KubeVirt so that vendors can expect the same
+control of their devices using Virtual Machines and Containers.
+
+## Goals
+
+- Align on the API changes needed to consume DRA enabled devices in KubeVirt
+- Align on how KubeVirt will consume devices by external DRA drivers
+
+## Non Goals
+
+- Replace device-plugin support in KubeVirt
+- Align on what drivers KubeVirt will support in tree
+- Address any complexity issues when DRA APIs in KubeVirt
+
+## Definition of Users
+
+- A user is a person that wants to attach a device to a VM
+- An admin is a person who manages the infrastructure and decide what kind of devices will be exposed via DRA
+- A developer is a person familiar with CNCF ecosystem that can develop automation using the APIs discussed here
+
+## User Stories
+
+- As a user, I would like to use my GPU dra driver with KubeVirt.
+- As a user, I would like to use KubeVirt's default driver.
+- As a user, in heterogeneous clusters, i.e. clusters made of nodes with different hardware managed through DRA drivers,
+  I should be able to easily identify what hardware was allocated to the VMI.
+- As a developer, I would like APIs to be extensible so I can develop drivers/webhooks/automation for custom use-cases.
+- As a device-plugin author, I would like to have a well documented way, intuitive way to support devices in KubeVirt.
+
+## Use Cases
+
+### Supported Usecases
```

Comment:
While migration cannot be done today, let's have it as intention to support it in the future.

Reply:
updated.